src/www/ui/admin-dashboard.php

```diff
@@ -159,7 +196,6 @@ function DatabaseMetrics()
     $connection_count = $row['val'];
 
     /**** Active connection count ****/
-    $current_query = (strcmp($this->pgVersion['server'], "9.2") >= 0) ? "state" : "current_query";
```

Comment:
granted that this is an unused variable. But how about adding this feature later at some time, because the current query info could be useful for debugging as well.